test/config_templates/rabbitmq_policy.bash

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+# Setup policy for rabbitmq mirored queue cluster
+
+policies=("on-all ^on\. {\"ha-mode\":\"all\"}" "sku-all ^sku\. {\"ha-mode\":\"all\"}"  "graph-all ^graph\. {\"ha-mode\":\"all\"}" "waterline waterline {\"ha-mode\":\"all\"}")
+for i in "${policies[@]}"
+do
+    docker exec -t -i docker_rabbit_1 rabbitmqctl set_policy $i
```

Comment:
I' know you're de-duplicating, I'm not sure how other's feeling,  it looks below is more readable and easy maintained, How do you think? 

docker exec -t -i docker_rabbitmq_1 rabbitmqctl set_policy on-all "^on\." '{"ha-mode":"all"}'
docker exec -t -i docker_rabbitmq_1 rabbitmqctl set_policy sku-all "^sku\." '{"ha-mode":"all"}'
docker exec -t -i docker_rabbitmq_1 rabbitmqctl set_policy graph-all "^graph\." '{"ha-mode":"all"}'
docker exec -t -i docker_rabbitmq_1 rabbitmqctl set_policy waterline "waterline" '{"ha-mode":"all"}'